src/controllers/albumController.js

```diff
@@ -75,10 +135,20 @@ const getAlbumWithAllUsersWhoPurchasedIt = async (req, res) => {
   });
 };
 
+// question for Akos: how can we test this route in postman? it is set to show 10 results per page and don't know how to see ne 2nd the 3d pages..etcпш
 const getFilteredAlbums = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch first 10 albums in a database with price more than 0'
+
+     #swagger.responses[200] = {
+				description: 'Filtered by price more than 0 albums were successfully fetched.',
+        schema: { albums: [{ $ref: '#/definitions/Album' }] }
```

Comment:
This endpoint also returns a `count` field:

```suggestion
        schema: { albums: [{ $ref: '#/definitions/Album' }], count: 1 }
```